tests/core/pyspec/eth2spec/utils/bls.py

```diff
@@ -60,7 +65,10 @@ def Aggregate(signatures):
 
 @only_with_bls(alt_return=STUB_SIGNATURE)
 def Sign(SK, message):
-    return bls.Sign(SK, message)
+    if bls == py_ecc_bls:
+        return bls.Sign(SK, message)
+    else:
+        return bls.Sign(SK.to_bytes(32, 'big'), message)
```

Comment:
It seems that milagro_bls requires 32-byte privkey: https://github.com/apache/incubator-milagro-crypto-rust/blob/736cdcb541daaae840a8fbc23d888831b134709b/src/bls381/core.rs#L79-L81

Reply:
Yes, it was one of those differences I hit when first trying it, and I had to dive into the binding benchmark code for an example of the correct private key format. Just checking the module, and converting to 32 bytes, seemed like the right thing to do. But I'm not 100% it produces the same effective signature, we should check.